pycbc/events/single.py

```diff
@@ -158,43 +198,62 @@ def check(self, trigs, data_reader):
         # Apply cuts to trigs before clustering
         # Cut on snr so that triggers which could not reach newsnr
         # threshold do not have newsnr calculated
+        if 'psd_var_val' in trigs:
```

Comment:
See [here](https://github.com/gwastro/pycbc/blob/v2.1.3/pycbc/events/ranking.py#L64) for the prdvar conversions